app/vulncode_app/models.py

```diff
@@ -50,13 +50,16 @@ class VulnerabilityReference(models.Model):
     class Meta:
         unique_together = ('vulnerability', 'source', 'reference_id')
 
+    def __str__(self):
+        return self.source
+
 
 class ImpactedPackage(models.Model):
     """
     Relates a vulnerability to package(s) impacted by it.
     """
     vulnerability = models.ForeignKey('Vulnerability')
-    package = models.ForeignKey('Package')
+    package_fk = models.ForeignKey('Package')
```

Comment:
What do you mean by clash? can you be more specific? at the db level? in code? where? provide an example with some stack/error trace.
In any case, package_fk is not a good name.
If a name change is really need to avoid some collision, then this would be the ManyToMany field name in Package that would need to be changed instead. 

Or maybe what you are really returning in you API as a primary model is not a Package... but an ImpactedPackage instead which would be actually more natural, what do you think?